packages/types/src/sdk/licensing/quota.ts

```diff
@@ -54,7 +55,13 @@ export const isConstantQuota = (
   return quotaType === QuotaType.CONSTANT
 }
 
-export type PlanQuotas = { [ key in PlanType]: Quotas | undefined }
+export interface Minimums {
+  users: number
+}
+
+export type PlanMinimums = { [key in PlanType]?: Minimums}
```

Comment:
Will we have type plans without minimums? If not, the `?` should be removed to use type safety

Reply:
Yeah we will, the existing pro and team plans that are day pass based. They have no concept of minimum. There is some protection around retrieving the minimum for any plan type, where we'll throw if the minimum isn't found for the given plan type. We shouldn't ever hit this code path for those plans